vm_server/send/test_cases.py

```diff
@@ -0,0 +1,33 @@
+import test_server
+import filecmp
+
+
+def test_execute_macro():
+  """Test the execute macro action"""
+  execute = test_server.execute_commands(1)
+  output_path = "..\\test\\execute_macro\\output\\output.txt"
+  expected_output_path = "..\\test\\execute_macro\\expected_output\\expected_output.txt"
+  assert filecmp.cmp(output_path, expected_output_path)
+
+def test_screenshot():
+  """Test the word screenshot action"""
+  execute = test_server.execute_commands(2)
+  assert execute == True
+
+def test_timeout():
+  """Test the failure due to timeout action"""
+  execute = test_server.execute_commands(3)
+  assert execute == False
+
+def test_config_pair():
+  """Test if config pairs are being set"""
+  execute = test_server.execute_commands(4)
+  assert execute == True
```

Comment:
should we also test the output instead of just whether the command executed or not? Like whether stdout / stderr have the expected contents and any other output expected?

Reply:
Change has been made. It compares the expected output now.